TimoCloud-Universal/src/main/resources/velocity/velocity.toml

```diff
@@ -0,0 +1,94 @@
+# Config version. Do not change this
+config-version = "1.0"
+
+# What port should the proxy be bound to? By default, we'll bind to all addresses on port 25577.
+bind = "0.0.0.0:25577"
+
+# What should be the MOTD? This gets displayed when the player adds your server to
+# their server list. Legacy color codes and JSON are accepted.
+motd = "&3A Velocity Server"
+
+# What should we display for the maximum number of players? (Velocity does not support a cap
+# on the number of players online.)
+show-max-players = 500
+
+# Should we authenticate players with Mojang? By default, this is on.
+online-mode = true
+
+# Should we forward IP addresses and other data to backend servers?
+# Available options:
+# - "none":   No forwarding will be done. All players will appear to be connecting from the
+#             proxy and will have offline-mode UUIDs.
+# - "legacy": Forward player IPs and UUIDs in a BungeeCord-compatible format. Use this if
+#             you run servers using Minecraft 1.12 or lower.
+# - "modern": Forward player IPs and UUIDs as part of the login process using Velocity's
+#             native forwarding. Only applicable for Minecraft 1.13 or higher.
+player-info-forwarding-mode = "LEGACY"
+
+# If you are using modern IP forwarding, configure an unique secret here.
+forwarding-secret = "CmlJNM3GzwNx"
```

Comment:
Why are we using a default value for a secret?

Reply:
the secret is only used for modern forwarding mode. but we don't use modern because it only works for servers with 1.13 or higher.